modules/folder/variables.tf

```diff
@@ -89,12 +89,33 @@ variable "iam_additive_members" {
   nullable    = false
 }
 
+variable "iam_policy" {
```

Comment:
`iam_policy` is mutually exclusive with the other iam variables? If so, I think we should mention it the the description

Reply:
good point, addressed in the most recent commit